ui/pages/onboarding-flow/create-password/__snapshots__/create-password.test.js.snap

```diff
@@ -30,12 +30,13 @@ exports[`Onboarding Create Password Render should match snapshot 1`] = `
       </ul>
     </div>
     <h2
-      class="box box--margin-top-1 box--margin-bottom-1 box--flex-direction-row typography typography--h2 typography--weight-bold typography--style-normal typography--color-text-default"
+      class="box mm-text mm-text--heading-lg box--flex-direction-row box--color-text-default"
     >
       Create password
     </h2>
     <h4
-      class="box box--margin-top-1 box--margin-bottom-1 box--flex-direction-row typography typography--h4 typography--weight-normal typography--style-normal typography--align-center typography--color-text-default"
+      align="center"
```

Comment:
Need to update the `align` prop to `textAlign`

Reply:
Will do.